rook/XMLDiff.py

```diff
@@ -418,7 +459,7 @@ def diff(self):
               gold_root = alt_gold_root_list[0]
           if not self.__same:
             pass #already failed
-          elif 'unordered' in self.__options.keys() and self.__options['unordered']:
+          elif 'unordered' in list(self.__options) and self.__options['unordered']:
```

Comment:
I don't think this needs to be changed, but I am not quite sure why this change was made?  ( `'unordered' in self.__options`  would also have worked, the `list` is redundant.)

Reply:
I was originally getting a Pylint warning C0201 (https://pylint.readthedocs.io/en/latest/user_guide/messages/convention/consider-iterating-dictionary.html) for iterating through the keys with `.keys()`. but you're right that the `list` is redundant, I applied the suggested fix